common/djangoapps/auth_exchange/tests/mixins.py

```diff
@@ -0,0 +1,40 @@
+"""
+Mixins to facilitate testing OAuth connections to Django-OAuth-Toolkit
```

Comment:
Of course.  That's left over from an intermediate stage where The DOP implementations were handled by the base test classes, and we overrode them to get DOT testing working.  Fixed.